src/modules/LanItems.rb

```diff
@@ -430,7 +430,13 @@ def ReplaceItemUdev(replace_key, new_key, new_val)
       operator = new_key == "NAME" ? "=" : "=="
       current_rule = getUdevFallback
```

Comment:
yes it is. However, the code somehow relies on that and I don't plan to add another change into this PR. I don't want to hijack the PR to far from original issue ;-)